.github/workflows/veracode-scan.yml

```diff
@@ -19,10 +19,10 @@ jobs:
         java-version: '8'
         distribution: 'temurin'
 
-    - name: Build with Maven
+    - name: Build project
       run: mvn -B package -Dmaven.test.skip=true --file pom.xml
 
-    - name: Veracode Upload And Scan (Static Application Security Testing)
+    - name: Veracode upload and scan (SAST)
```

Comment:
minor: would you please consider including the complete name before (SAST), like what's done for SCA?
My concern is someone later wondering what SAST means.

Reply:
Sounds good, I will update the code.